module/Photo/src/Photo/Listener/AlbumDate.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Photo\Listener;
+
+/**
+ * Doctrine event listener class for Album and Photo entities.
+ * Do not instantiate this class manually.
+ */
+
+class AlbumDate
+{
+
+    public function prePersist($eventArgs)
+    {
+        $entity = $eventArgs->getEntity();
+        if ($entity->getResourceId() === 'album') {
+            $this->albumPersisted($entity);
+        } elseif ($entity->getResourceId() === 'photo') {
+            $this->photoPersisted($entity);
+        }
```

Comment:
Is het hier niet beter om `instanceof` te gebruiken om te checken wat voor entity het precies is? Het hoeft namelijk niet zo te zijn dat elke entity de `getResourceId()` methode implementeert.